lib/src/Base/Algo/openturns/ApproximationAlgorithmImplementation.hxx

```diff
@@ -144,6 +149,8 @@ private:
   /** Verbosity flag */
   Bool verbose_;
 
+  /** isModelSelection flag */
+  Bool isModelSelection_;
```

Comment:
If the `isModelSelection` method returns a hard-coded value, this attribute can be removed.
```suggestion
```